src/main/java/de/tum/in/www1/artemis/service/BuildLogEntryService.java

```diff
@@ -100,14 +98,24 @@ private boolean isInfoLog(String log) {
         return (log.startsWith("[INFO]") && !log.contains("error")) || log.startsWith("[INFO] Downloading") || log.startsWith("[INFO] Downloaded");
     }
 
+    private boolean isGradleInfoLog(String log) {
+        return log.startsWith("Downloading https://services.gradle.org") || log.startsWith("...........10%") || log.startsWith("Here are the highlights of this release:")
+                || log.startsWith("- ") || log.startsWith("For more details see") || log.startsWith("Starting a Gradle Daemon");
```

Comment:
```suggestion
        return GRADLE_LOG_PREFIXES.stream().anyMatch(prefix -> log.startsWith(prefix));
```
```java
private static final GRADLE_LOG_PREFIXES = List.of(…);
```

There are a lot of checks that all check for `log.startsWith(…)`. Using a constant list would simplify the condition and make it more extensible in the future.

Even if is has to be combined with e.g. `log.contains("…") || PREFIXES.stream().anyMatch(…)` the readability of the condition gets improved.